risc0/zkvm/src/guest/env.rs

```diff
@@ -109,6 +109,7 @@ pub fn syscall(syscall: SyscallName, to_host: &[u8], from_host: &mut [u32]) -> s
 /// In order to be valid, the [crate::Receipt] must have `ExitCode::Halted(0)`, an
 /// empty assumptions list, and an all-zeroes input hash. It may have any post
 /// [crate::SystemState].
+#[must_use]
```

Comment:
My understanding is the `Result` is already marked as `#[must_use]` and so this is redundant. I also thought there was a clippy lint for this:
https://std-dev-guide.rust-lang.org/policy/must-use.html

Reply:
https://rust-lang.github.io/rust-clippy/stable/index.html#/double_must_use

Reply:
Oh, nice! I did not know that. Removed the new `#[must_use]` attributes in 444d8710d50e809bde089f830bddb0eebbf9be4c